crypto/s2n_certificate.c

```diff
@@ -834,6 +834,25 @@ int s2n_cert_get_x509_extension_value_length(struct s2n_cert *cert, const uint8_
     return S2N_SUCCESS;
 }
 
+int s2n_cert_get_x509_extension_value_length_exists(struct s2n_cert *cert, const uint8_t *oid,
+                                                    uint32_t *ext_value_len, bool *is_present)
+{
+    POSIX_ENSURE_REF(cert);
+    POSIX_ENSURE_REF(oid);
+    POSIX_ENSURE_REF(ext_value_len);
+
+    *is_present = true;
```

Comment:
`is_present` needs a ensure ref. That or only set the value if it's not null. The advantage there would be callers that don't care about that information would be able to pass `NULL` and not have to provide a bool address. I'd be fine with either, though.